spec/models/journey_spec.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Journey, type: :model do
+  it { is_expected.to belong_to(:move) }
+  it { is_expected.to belong_to(:supplier) }
```

Comment:
should not be the move linked with the supplier?

Reply:
as discussed, it probably should be linked, but not as part of this work :-)